libs/abseil-cpp/Makefile

```diff
@@ -35,17 +35,22 @@ endef
 CMAKE_INSTALL:=1
 
 CMAKE_HOST_OPTIONS += \
-	-DCMAKE_CXX_STANDARD=14 \
+	-DABSL_PROPAGATE_CXX_STD=ON \
 	-DABSL_ENABLE_INSTALL=ON \
 	-DABSL_USE_GOOGLETEST_HEAD=OFF
 
 CMAKE_OPTIONS += \
-	-DCMAKE_CXX_STANDARD=14 \
+	-DABSL_PROPAGATE_CXX_STD=ON \
 	-DABSL_ENABLE_INSTALL=ON \
 	-DABSL_USE_GOOGLETEST_HEAD=OFF
 
 TARGET_CFLAGS += $(FPIC)
 
+define Package/abseil-cpp/install
+	$(INSTALL_DIR) $(1)/usr/lib
+	$(CP) $(PKG_INSTALL_DIR)/usr/lib/* $(1)/usr/lib/
```

Comment:
Instead of `CP` you should use one of:

```Makefile
INSTALL_BIN:=install -m0755
INSTALL_SUID:=install -m4755
INSTALL_DIR:=install -d -m0755
INSTALL_DATA:=install -m0644
INSTALL_CONF:=install -m0600
```

And it might be a good idea to specify which libraries you expect to copy instead of a wildcard.